### ISSUE TITLE
Modify/Cleanup template files after `PAM50_cord` is updated

### ISSUE THREAD

**Issue by jcohenadad** [spinalcordtoolbox/PAM50#25]
After modifying the spinal cord segmentation in https://github.com/spinalcordtoolbox/PAM50/pull/21, we need to consider updating/removing:
- PAM50_levels_continuous.nii.gz,
- PAM50_csf.nii.gz,
- PAM50_centerline.nii.gz

As per https://github.com/spinalcordtoolbox/PAM50/pull/21#issuecomment-1731586138

**Comment by joshuacwnewton** [comment]
I don't think we ever followed up on this -- I noticed that e.g. there are presently gaps between CSF/cord:

<img width="346" height="175" alt="Image" src="https://github.com/user-attachments/assets/41e4bcaa-fa30-410f-bbfb-bf09ad0af93e" />
